cukinia/common_security_tests.d/apt.conf.j2

```diff
@@ -174,7 +174,7 @@ $ESSENTIAL_PACKAGES_DEBOOTSTRAP\
 UNRECOGNIZED_PACKAGES=""
 
 for package in $(apt list --installed 2>/dev/null | awk -F '/' 'NR>1 {print $1}'); do
-    if ! apt-cache rdepends --recurse "$package" | grep -Eq "^(  )?${ESSENTIAL_PACKAGES}$"; then
+    if ! apt-cache rdepends --installed --recurse "$package" | grep -Eq "^(  )?${ESSENTIAL_PACKAGES}$"; then
```

Comment:
Can you explain why you need to change that?

Reply:
sure. While it does not seem to pose a problem during CI (when the server is newly installed), when you run "apt-cache rdepends" after having run a "apt update", then the result is completely different because it seems to return dependency for everything, not just installed package.
The idea of the test being that we want to know whether there is an installed package justifying the presence of another, then if we want the test to work even after an "apt update", then we must tell apt-cache to only look in installed packages.